src/renderable.jl

```diff
@@ -166,3 +166,18 @@ end
 
 # render nothing by not doing anything
 add_renderable!(rm::RenderModel, ::Nothing) = rm
+
+"""
+Render function for text
+"""
+function add_renderable!(rm::RenderModel, t::String)
+    font_size = 12
+    x = 10
+    y = 1.5*font_size
+    y_jump = 1.5 * font_size
+    for line in split(t, '\n')
+        add_instruction!(rm, render_text, (line, x, y, font_size, colorant"gray75"), coordinate_system=:camera_pixels)
+        y += y_jump
+    end
+    return rm
+end
```

Comment:
I might have merged too quickly, there are a lot of magic numbers in there, what is that function for?
@mattuntergassmair